tests/README.md

```diff
@@ -0,0 +1,2 @@
+These are smoke tests for riscv64. They are set up to be run via qemu on the
+host machine and require LLVM testing tools and iree-import-tflite.
```

Comment:
wrong folder for this README (move down a level to tests/riscv64)

Reply:
Whoops, thanks for catching. I blame vscode